wolfssl/internal.h

```diff
@@ -3465,6 +3465,7 @@ struct WOLFSSL {
     unsigned long    peerVerifyRet;
     byte             readAhead;
     byte             sessionCtxSz;       /* size of sessionCtx stored */
+    WOLFSSL_BIO      *bio;               /* BIO tied with SSL */
```

Comment:
What is the reason for adding this to the WOLFSSL structure? If trying to use a BIO instead of socket, biord and biowr can be used.